kateryna-tolochko/README.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started with AliExpress
+
+AliExpress is a popular online marketplace where you can find a wide variety of products at affordable prices. Whatever you are looking for, AliExpress offers a huge selection from sellers around the world. If you are new to AliExpress, this guide will help you get started and make the most of your shopping experience.
+
+## Step 1: Create an Account
+
+1. Visit the [AliExpress website](https://www.aliexpress.com/).
+
+2. Find **Join Free** button and click it.
```

Comment:
Currently, there is no button 'Join Free' on the website. I see either 'Register' or 'Sign in' buttons in the 'Account' menu. See 
![image](https://github.com/ivancheban/kpi-tech-writer-course/assets/57861402/092aabac-57b9-4e07-99a0-08e677871032)
and 
![image](https://github.com/ivancheban/kpi-tech-writer-course/assets/57861402/98085cbe-4ccf-491a-b781-399fdba4bd54)

Please rewrite this part in accordance with the current UI.